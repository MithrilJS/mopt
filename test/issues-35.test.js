"use strict";

var code = require("./lib/code");

describe("Issues", () => {
    describe("#35 hyphenated attributes", () => {
        it("should be supported in the selector", () =>
            expect(
                code(`m(".fooga[wooga-booga=1]")`)
            )
            .toMatchSnapshot()
        );
        
        it("should be supported as an attribute", () =>
            expect(
                code(`m(".fooga", { "wooga-booga" : 1 })`)
            )
            .toMatchSnapshot()
        );
    });
});
