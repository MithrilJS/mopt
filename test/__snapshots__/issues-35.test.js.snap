// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Issues #35 hyphenated attributes should be supported as an attribute 1`] = `"m.vnode(\\"div\\",undefined,{className:\\"fooga\\",\\"wooga-booga\\":1},[],undefined,undefined);"`;

exports[`Issues #35 hyphenated attributes should be supported in the selector 1`] = `"m.vnode(\\"div\\",undefined,{className:\\"fooga\\",\\"wooga-booga\\":\\"1\\"},[],undefined,undefined);"`;
